Allow ongoing checks (#35)
adRise/update-pr-branch: README.md

Default: true

> The action will skip PRs that have failed checks. Please note that if `allow_ongoing_checks` is not set to `true`, the action will skip PRs with ongoing checks. This may result in the failure to update PR branches when the action is triggered while checks for those PRs are still in progress.

Reviewer: ```suggestion
The action will skip updating PRs that have failed checks. Please note that if `allow_ongoing_checks` is set to `false`, the action will skip updating PRs with ongoing checks. This will result in the failure to update PR branches when the action is triggered while checks for those PRs are still in progress.
```

Author: thanks

### `allow_ongoing_checks`

**Optional**

Default: false

> The action will consider PRs that have ongoing checks. This is useful when the action is triggered while checks for some otherwise qualified PRs are still in progress.

Reviewer 2: This option only works if `require_passed_checks` is `true` (even though it's the default value). We can add a short comment here.

Author: Thanks
